cypress/integration/Pill.spec.ts

```diff
@@ -0,0 +1,116 @@
+import * as h from '../helpers';
```

Comment:
Can we rename this file to `PillPreview` that way it indicates it's testing the pill in preview!

Reply:
Agreed

Reply:
Done